src/lib/utils.ts

```diff
@@ -120,6 +120,9 @@ export interface ParsedName {
   title: string
 }
 
+/**
+ * @deprecated AutoForm vracia udaje uz rozparsovane 
```

Comment:
Ak sa `parseFullName` uz nikde nepouyziva tak kludne zmazat 👍

Reply:
este sa pouziva v osobnych udajoch pre DP preto som dal iba deprecated zatial a ked sa bude upravovat ta cast s DP tak by sa to malo odstranit, snazil som sa menit co najmenej kodu pre tento fix kedze to potrebujeme ASAP